src/main/java/at/hannibal2/skyhanni/config/features/Inventory.java

```diff
@@ -125,18 +125,53 @@ public static class SackDisplay {
         public boolean enabled = true;
 
         @Expose
-        @ConfigOption(name = "Number Format", desc = "Either show Default, Formatted or Unformatted numbers." +
+        @ConfigOption(name = "Number Format", desc = "Either show Default, Formatted or Unformatted numbers.\n" +
                 "§eDefault: §72,240/2.2k\n" +
                 "§eFormatted: §72.2k/2.2k\n" +
                 "§eUnformatted: §72,240/2,200")
```

Comment:
Should this line be 
"§eUnformatted: §72,240/2,240"
or am i missing something?

Reply:
I can't get the exact capacity of sacks, i'm just formatting what hypixel give me (2.2k here)
- 2,240/2.2k -> thats what hypixel display
- 2.2k/2.2k -> everything is formatted, so 2,240 give 2.2k and i don't touch sack capacity
- 2,240/2,200 -> i only format sack capacity (2.2k) so it give 2,200

That's just so users can see what they gonna get in the display, i can change to 2240/2240, does not really matter